src/main/java/frc/robot/subsystems/Arm/ArmIOReal.java

```diff
@@ -40,4 +40,8 @@ public void update()
     
   }
 
+  public boolean isEncoderConnected() {
+    return m_armEncoder.isConnected();
```

Comment:
Per other comments below there should be a warning here to inform the user the robot is in a bad state.